crux/src/Crux/UI/JS.hs

```diff
@@ -3,33 +3,61 @@
 -- | Utilites for generating JSON
 module Crux.UI.JS where
 
-import Data.Text(unpack)
+import Data.Text(unpack, Text)
+import qualified Data.Text as Text
+import Numeric
 import Data.List(intercalate)
 import Data.Maybe(fromMaybe)
 import System.Directory( canonicalizePath )
 
+import Data.Aeson.Encode.Pretty (encodePretty)
+import Data.Aeson (decode, Value)
+import qualified Data.ByteString.Lazy.Char8 as BL
+
+
 import What4.ProgramLoc
 
+-- |`jsLoc` takes a program location, and optionally returns it
+-- as `JS String`
```

Comment:
Haddock formatting nit picks:

```suggestion
-- | 'jsLoc' takes a program location, and optionally returns it
-- as @JS String@.
```

(In Haddock, `@...@` delimits text that should be rendered in monospace font, and `'...'` is a stronger version of `@...@` that attempts to link to a Haskell identifier with a hyperlink.)

Reply:
Done